Source/Models/Token.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 public class Token {
-    let accessToken: String
+    public let accessToken: String
```

Comment:
Nice catch, didn't think of this :P

The `LockRequest` constructor requires an access token when initiating. I'm thinking that this request should instead require a `Token` instance. This way it can extract the accesstoken when needed, and also verify that the token hasn't expired. If it has expired, it should be able to refresh the token without having to fail the request.

But what I'm describing above is not related to this PR, now that I think about it. I'll create a second PR once this has been merged with these fixes.